frontend/.npmrc

```diff
@@ -0,0 +1,2 @@
+node-options=--openssl-legacy-provider
+node-linker=hoisted
```

Comment:
I add this to address [building issues on Linux](https://github.com/pnpm/pnpm/issues/3585).

Feel free to change it if you have a better solution. :D